local_runner/python/datastore.py

```diff
@@ -101,12 +101,22 @@ def fetch(self, limit=None, offset=None):
             result = result[:limit]
 
         return result
+    
+    def cmp(self, a, b):
+        return (a > b) - (a < b)
 
     def _make_orderer(self):
-        return lambda ent: tuple(
-            (-1 if o[0] == "-" else 1) * getattr(ent, o.strip("+-"), 0)
-            for o in self.order
-        )
+        def order_func(item):
+            sort_keys = []
+            for order_key in self.order:
+                desc = order_key.startswith('-')
+                key = order_key[1:] if desc else order_key
```

Comment:
this does not work for `"+x"` keys

Reply:
According to the docs, you only use `-`, ascending keys are without `+`.

Link: https://cloud.google.com/datastore/docs/concepts/queries#sort_orders